.github/workflows/ci.yml

```diff
@@ -13,9 +13,22 @@ jobs:
       run: stylua --check .
   Tests:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          # For all versions see https://github.com/neovim/neovim/tags
+          - neovim: 0.8.3
+          - neovim: 0.9.4
+          - neovim: nightly
```

Comment:
On a second thought, the stability of the pipeline should not depend on an unstable Neovim branch that isn't even tagged yet (nighly). wdyt @peterfication ?

Reply:
Yeah, it might have been too much. I was just amazed that we could easily test against nightly as well. But it's not really necessary, especially not for this plugin which doesn't do anything on the edge of Neovim.